mappings/net/minecraft/util/collection/IdList.mapping

```diff
@@ -1,9 +1,14 @@
-CLASS net/minecraft/class_2361 net/minecraft/util/IdList
+CLASS net/minecraft/class_2361 net/minecraft/util/collection/IdList
 	FIELD field_11098 list Ljava/util/List;
 	FIELD field_11099 nextId I
 	FIELD field_11100 idMap Ljava/util/IdentityHashMap;
+	METHOD <init> (I)V
+		ARG 1 ids
```

Comment:
Why is `ids` a primitive int? I expect an array or a list. Is this `size` or `maxId`?

Reply:
Either I misread, or Enigma messed up. Changed to `initialSize`

Reply:
Either I misread, or Enigma messed up. Changed to initialSize